share/revng/cmake/TupleTreeGenerator.cmake

```diff
@@ -383,12 +390,18 @@ function(
     list(APPEND SCALAR_TYPE_ARGS --scalar-type "'${ET}'")
   endforeach()
 
+  # We add the `mixins` parameter only when processing the model
+  set(PYTHON_MIXINS_ARGS)
+  if(${NAMESPACE} STREQUAL "model")
+    list(APPEND PYTHON_MIXINS_ARGS --mixins "${PYTHON_MIXINS}")
+  endif()
```

Comment:
No special-casing the model, if `PYTHON_MIXINS` is specified it needs to be added